packages/api/lib/granuleUtils.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const { parseS3Uri } = require('@cumulus/common/aws');
+
+const getKeyFromFile = (file) => {
+  if (file.filename) return parseS3Uri(file.filename).Key;
```

Comment:
If this is being removed, why are we still checking for it?

Reply:
I think I see. This is a temporary measure as part of the converting files from a cumulus message to the new API format.

What will still have the `filename` property that makes this code necessary?

Reply:
I don't know if this is the reason, but we have some tests that expect this value to still be alive. There might also be some fixtures/ancillary setup data that use it.

Reply:
This comment talks about why there are still instances of `filename`: https://github.com/nasa/cumulus/pull/800#issuecomment-462491409

That being said, I've taken a closer look at where this instance of `filename` is used, and I _think_ it can be removed.  Waiting on one last test build ...

Reply:
And ... yeah, we still need `filename` here.  The `filename` field seems to be the only reliable way to determine a file's key from the output of the `SyncGranule` task, which is needed by `packages/api/es/indexer.js`.